packages/editor/src/lib/exports/domUtils.ts

```diff
@@ -0,0 +1,46 @@
+export function getRenderedChildNodes(node: Element): Iterable<Node> {
+	if (node.shadowRoot) {
+		return node.shadowRoot.childNodes
+	}
+	if (isShadowSlotElement(node)) {
+		const assignedNodes = node.assignedNodes()
```

Comment:
a comment here would be good, i don't think many of use are very familiar with shadow dom and what this is doing 🙃